book/forms.rst

```diff
@@ -667,7 +674,7 @@ boxes. However, the :doc:`date field </reference/forms/types/date>` can be
 configured to be rendered as a single text box (where the user would enter
 the date as a string in the box)::
 
-    ->add('dueDate', 'date', array('widget' => 'single_text'))
+    ->add('dueDate', Type\DateType::class, array('widget' => 'single_text'))
```

Comment:
Linebreaks aren't needed when we don't cross the 85 characters in a code example (otherwise, it'll cause horizontal scrollbars). The Symfony standard is to never use line wrapping, so I only want to use line breaks when needed to avoid scrollbars.